app/models/document.rb

```diff
@@ -239,7 +239,7 @@ def publish!
         email_alert_api.send_alert(EmailAlertPresenter.new(self).to_json)
       end
 
-      publish_request.code == 200 && rummager_request.code == 200
+      publish_request.code == 200 && rummager_request.code == 202
```

Comment:
Being so prescriptive on the expected return type feels very brittle (eg if Rummager or Publishing API suddenly starts returning another 2XX status code, this will break). I believe that `gds-api-adapters` will raise an exception if there's a non-200 return code, so so is this line is even needed?

Thoughts? @danielroseman @tijmenb